cmd/did-method-cli/internal/configcommon/helpers_test.go

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package configcommon
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/square/go-jose/v3"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	configData = `{
+  "consortiumData": {
+    "domain": "consortium.net",
+    "genesisBlock": "6e2f978e16b59df1d6a1dfbacb92e7d3eddeb8b3fd825e573138b3fd77d77264",
+    "policy": {
+      "cache": {
+        "maxAge": 2419200
+      },
+      "numQueries": 2,
+      "historyHash": "SHA256"
+    }
+  },
+  "membersData": [
+    {
+      "domain": "stakeholder.one",
+      "policy": {"cache": {"maxAge": 604800}},
+      "endpoints": [
+        "http://endpoints.stakeholder.one/peer1/",
+        "http://endpoints.stakeholder.one/peer2/"
+      ],
+      "privateKeyJwkPath": "%s"
+    }
+  ]
+}`
+
+	jwkData = `{
+        "kty": "OKP",
+        "kid": "key1",
+        "d": "-YawjZSeB9Rkdol9SHeOcT9hIvo_VuH6zM-pgtk3b10",
+        "crv": "Ed25519",
+        "x": "bWRCy8DtNhRO3HdKTFB2eEG5Ac1J00D0DQPffOwtAD0"
+      }`
+)
+
+func TestWriteConfig(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
+		filesData := map[string][]byte{
+			"abc": []byte("data data"),
+			"def": []byte("data2 data2"),
+		}
+
+		dir, err := ioutil.TempDir("", "")
+		require.NoError(t, err)
+
+		defer func() { require.NoError(t, os.RemoveAll(dir)) }()
+
+		require.NoError(t, WriteConfig(dir, filesData))
+
+		_, err = os.Stat(dir + "/did-trustbloc/abc.json")
+		require.False(t, os.IsNotExist(err))
+
+		_, err = os.Stat(dir + "/did-trustbloc/def.json")
+		require.False(t, os.IsNotExist(err))
+	})
+}
+
+func TestGetConfig(t *testing.T) {
+	t.Run("fail: missing arg", func(t *testing.T) {
+		_, err := GetConfig(&cobra.Command{})
+		require.Error(t, err)
+	})
+
+	t.Run("fail: can't read config file", func(t *testing.T) {
+		os.Clearenv()
+		require.NoError(t, os.Setenv(ConfigFileEnvKey, "bad.file"))
+
+		_, err := GetConfig(&cobra.Command{})
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed to read")
```

Comment:
Add check for config as well (should be nil) don't use _.  This comment applies to all error tests.